exercises/practice/raindrops/.meta/create_test_table

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
```

Comment:
I like to have automation as early as possible in the track 👍

Reply:
Is it feasible to have it general enough to put a version in the ./bin directory?

Reply:
Possibly. We would need to templatize the columns. We could auto generate the schema, both files and maybe more. It might be good to write a specific generator or three first to have more samples then try to make a generic generator that can everything we need. 

Depending on how complicated things get, bash may or may not suffice. I can attempt bash, awk, sed, jq but if that gets tricky, Python is my wheelhouse and what I would easiest be able to use.